lib/dogma/rules.ex

```diff
@@ -28,6 +29,9 @@ defmodule Dogma.Rules do
     Application.get_env :dogma, :rule_set, @default_rule_set
   end
 
+  defp async_test_script(script, formatter, rule_set) do
```

Comment:
This function doesn't actually do anything. Can we remove it and just use `Task.async/1` directly in the `Enum.map/2`?